### PR TITLE
Update kubernetes example

### PR DESCRIPTION
**Pull request by thompsonja**
@enisoc

### CODE REVIEW

File: examples/kubernetes/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ You'll need [Go 1.3+](http://golang.org/doc/install) in order to build the
 ### Build and install vtctlclient
 
 ```
-$ go get github.com/youtube/vitess/go/cmd/vtctlclient
+vitess/examples/kubernetes$ go build ../../go/cmd/vtctlclient
```
Comment (enisoc) [inline examples/kubernetes/README.md]: Why the change? The "go get" one still works for me in a clean Docker image, and I think it's more self-contained because you don't have to download anything in advance.

Comment (thompsonja) [inline examples/kubernetes/README.md]: Really?  I get the following:
# github.com/youtube/vitess/go/stats/influxdbbackend
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:40: undefined: client.ClientConfig
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:60: undefined: client.Series
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:62: undefined: client.Series
../../gopath/src/github.com/youtube/vitess/go/stats/influxdbbackend/influxdb_backend.go:73: backend.client.WriteSeries undefined (type *client.Client has no field or method WriteSeries)

Comment (thompsonja) [inline examples/kubernetes/README.md]: Damnit markdown!

Comment (enisoc) [inline examples/kubernetes/README.md]: Didn't you remove the influxdb plugin from vtctlclient to fix that?

Comment (thompsonja) [inline examples/kubernetes/README.md]: I thought it would but I still get that error.  Removing influxdb plugin from vtctlclient at least allows for running go build in the vtctlclient folder.

Comment (thompsonja) [inline examples/kubernetes/README.md]: Yep, as discussed offline it was just an issue of leftover files from previous calls to go get (prior to my earlier fix).  Reverted the change.

**Comment by enisoc** [comment]
LGTM thanks for the updates. :)
